tests/test_schema_validation.py

```diff
@@ -43,10 +43,10 @@ def check_validation_errors(filename, expected_validation_error_messages):
 
     assert schema_errors == {}
 
-    assert len(validation_errors) == len(expected_validation_error_messages)
```

Comment:
This will not longer ensure that there are no extra validation errors?

Reply:
yeh I switched for loop to loop over the expected error messages and removed this so it gave a better error message rather than expected 5 got 6.
Ill add this back in under the loop.